src/components/my-reservations/myReservations.jsx

```diff
@@ -0,0 +1,66 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchReservations } from '../../redux/myReservations/myReservationsSlice';
+import { fetchTutories } from '../../redux/deleteClass/deleteClassSlice';
+import './myReservations.css';
+
+const MyReservations = () => {
+  const dispatch = useDispatch();
+  const tutories = useSelector((state) => state.deleteClass.tutories);
+  const reservations = useSelector((state) => state.myReservations.reservations);
+
+  useEffect(() => {
+    dispatch(fetchTutories());
+    dispatch(fetchReservations());
+  }, [dispatch]);
+
+  const combinedData = tutories.map((tutory) => {
+    const reservation = reservations.find((reservation) => reservation.classSubject_id === tutory.id);
+    return {
+      id: tutory.id,
+      subject: tutory.subject,
+      description: tutory.description,
+      date: reservation ? reservation.date : '',
+      city: reservation ? reservation.city : '',
+    };
+  });
```

Comment:
When I open the My Reservations page, It displays all classes if the one that were not reserved.
The page should only display reservations.

As you can see, I only have one reservation inserted in the database.
<img width="306" alt="Screen Shot 2023-06-21 at 16 22 43" src="https://github.com/enoqueJonas/tutoring-app-frontend/assets/76396858/94c77cff-c9a1-46fe-884b-de322e100a54">

But it's displaying all the classes I have inserted.
<img width="1085" alt="Screen Shot 2023-06-21 at 16 23 07" src="https://github.com/enoqueJonas/tutoring-app-frontend/assets/76396858/4ca5dc01-b2a0-4398-af77-9223d6624f43">

Kindly fix the issue.

Reply:
@enoqueJonas , thanks a lot for the review, and sorry for the oversight. Will fix the issue ASAP